VultisigApp/VultisigApp/Views/Vault/CoinDetailView.swift

```diff
@@ -46,8 +46,14 @@ struct CoinDetailView: View {
         .navigationBarBackButtonHidden(true)
         .navigationTitle(NSLocalizedString(coin.ticker, comment: ""))
         .onAppear {
+            sendTx.coin = coin
```

Comment:
Do we need to set coin here? `.reset` is setting coin

Reply:
Changed to use reset only on change and it also works. Thanks.